src/renderer/features/wallet-select/service/__tests__/walletSelectService.test.ts

```diff
@@ -0,0 +1,24 @@
+import { type Wallet, WalletType } from '@/shared/core';
+import { walletSelectService } from '../walletSelectService';
+
+describe('walletSelectService', () => {
+  const wallets = [
+    { id: 1, type: WalletType.POLKADOT_VAULT, name: 'pv' },
+    { id: 2, type: WalletType.WALLET_CONNECT, name: 'wc' },
+    { id: 3, type: WalletType.SINGLE_PARITY_SIGNER, name: 'sps' },
+  ] as Wallet[];
+
+  test('should group wallets POLKADOT_VAULT > MULTISIG > NOVA_WALLET > WALLET_CONNECT > WATCH_ONLY > PROXIES', () => {
```

Comment:
```suggestion
  test('should group wallets POLKADOT_VAULT > MULTISIG > NOVA_WALLET > WALLET_CONNECT > WATCH_ONLY > PROXIEDS', () => {
```

As I see this test doesn't check all wallet types, only `PV`, `WC` and `PV (single parity signer)`